src/main/resources/application.conf

```diff
@@ -550,7 +550,36 @@ mantis {
 
       # Force checksum verification of all data that is read from the file system on behalf of a particular read
       verify-checksums = true
+
+      # This ensures that only 32 open files can be accessed at once
+      max-open-files = 32
```

Comment:
Why `max-open-files = 32` for leveldb ? Up until now we were using default (`1000`) and everything was ok with it.